tests/unit/type/string.tcl

```diff
@@ -348,6 +348,17 @@ start_server {tags {"string"}} {
         assert_equal 0 [r getbit mykey 10000]
     }
 
+    test "GETBIT pos larger than UINT_MAX" {
+        r del mykey
+        r setbit mykey [expr (1 << 32) - 1] 1
+        set oldval [lindex [r config get proto-max-bulk-len] 1]
+        r config set proto-max-bulk-len [expr (1 << 29) + 1]
+        r append mykey "\xFF"
+        set res [r getbit mykey [expr (1 << 32)]]
+        r config set proto-max-bulk-len $oldval
+        assert_equal 1 $res
+    }
```

Comment:
i'm not sure about adding that test.. it consumes a lot of memory, which means the test suite can't run on lean machines.
maybe we can drop it, or in some way make it skipped by default...

Reply:
@huangzhw i thought of several solutions, and consulted Yossi. the one we think is right is to add a `large-memory` tag to the test, and then maybe add `denytags` argument to the freebsd and valgrind CI.
can you handle that?

Reply:
I will try to do it. But I don't like tcl too, I never learn it. So maybe it will take some time.

Reply:
there's no TCL coding.
just terminate the `if` statement above with a tag:
```diff
-    }
+    } {} {large-memory}
```

add `--tags -large-memory` to the daily.yml when it runs the test suite in the valgrind job.
and maybe also ducment it in `tests/README.md`

Reply:
Thanks @oranagra.
I found more places like `bitcount` and `bitpos`.
So I think I should read this file carefully.

Reply:
> just terminate the `if` statement above with a tag

Is the tag on the `test` as in `test { ... } {} {large-memory}`?

Reply:
yes, Yossi recently extended the tagging system in #9033

Reply:
He's a great TCL programmer. :-)

Reply:
I'm a python programmer. regrettably redis has no python.

Reply:
I think none of us had ever used TCL before we had to look at Redis tests.

Reply:
totally agree. However there is no documentation to describe the test.